src/Http/Http.Extensions/src/RequestDelegateFactory.cs

```diff
@@ -672,52 +806,10 @@ private static Expression BindParameterFromValue(ParameterInfo parameter, Expres
             var parameterNameConstant = Expression.Constant(parameter.Name);
             var sourceConstant = Expression.Constant(source);
 
-            if (parameter.ParameterType == typeof(string))
+            if (parameter.ParameterType == typeof(string) ||
+                parameter.ParameterType == typeof(IFormFileCollection))
```

Comment:
```suggestion
            if (parameter.ParameterType == typeof(string))
```

BindParameterFromValue isn't called for IFormFileCollections, right?

Reply:
I'll double-check before I push up a rebase, but (at least initially when the PR was newish) this was to be able to re-use as much of the code here as I could to reduce duplication, so it was going in here to bind the value.

Reply:
You're right - this is redundant now.